velox/functions/prestosql/aggregates/ChecksumAggregate.cpp

```diff
@@ -77,18 +77,36 @@ class ChecksumAggregate : public exec::Aggregate {
     }
   }
 
+  bool isNullOrNullArray(const TypePtr& type) {
+    if (type->isUnKnown()) {
+      return true;
+    }
+    // only support null array type for now.
```

Comment:
If it a sentence (ends with `.`). It should be capitalized.

```suggestion
    // Only supports null array type for now.
```